cpp/include/proton/delivery.hpp

```diff
@@ -64,6 +65,9 @@ class delivery : public transfer {
     /// Settle with MODIFIED state.
     PN_CPP_EXTERN void modify();
 
+    /// Return the tag for this delivery.
+    PN_CPP_EXTERN binary tag() const;
```

Comment:
You should rebase your PR branch on top of `main`, to get rid of this (already merged) code. Or you can just delete the commit from the branch; whichever you find simpler.

Reply:
Just did that. Thanks.